code/modules/urist/modules/shipbattles/weapons.dm

```diff
@@ -282,6 +282,112 @@
 	desc = "It's a big warhead for a big torpedo. Shove it in a torpedo casing and you've got yourself a torpedo." //torpedo
 	icon = 'icons/urist/items/ship_projectiles.dmi'
 	icon_state = "torpedowarhead"
+	var/safety = 1 //Integrated safeties are functioning as intended. This thing's inert.
+	var/is_rigged = 0 //Has this thing been jerry-rigged?
+	var/riggedstate = 0 //What stage of rigging are we in?
+	var/obj/item/device/attached_device //Whatever is attached that'll detonate us.
+
+/obj/item/shipweapons/torpedo_warhead/examine(mob/user)
+	..(user)
+	switch(riggedstate)
+		if(1) to_chat(user, "<span class='notice'>It's control circuitry is exposed.</span>")
+		if(2) to_chat(user, "<span class='notice'>It's control circuitry is exposed, and the internal wiring appears to have been modified.</span>")
+		if(3) to_chat(user, "<span class='notice'>It's control circuitry is exposed, the internal wiring appears to have been modified, and a wire has been rigged to the detonator circuit.</span>")
+	switch(safety)
+		if(0) to_chat(user, "<span class='warning'>The safeties have been disabled.</span>")
+		if(1) to_chat(user, "<span class='notice'>The safeties are enabled.</span>")
+	if(is_rigged)
+		to_chat(user, "<span class='warning'>There is a [attached_device] attached to the warhaed.</span>")
+
+/obj/item/shipweapons/torpedo_warhead/attackby(var/obj/item/I, mob/user as mob)
+	if(istype(I, /obj/item/weapon/crowbar))
+		if(riggedstate == 1 && !attached_device) // can't close it if it's got something it's not supposed to have.
+			to_chat(user, "<span class='notice'>You carefully close the warhead's circuitry panel.</span>")
+			riggedstate = 0
+			playsound(src.loc, 'sound/items/Crowbar.ogg', 50, 1)
+		else if(!riggedstate)
+			to_chat(user, "<span class='notice'>You carefully lever open the warhead's circuitry panel.</span>")
+			riggedstate++
+			playsound(src.loc, 'sound/items/Crowbar.ogg', 50, 1)
+		else
+			to_chat(user, "<span class='notice'>You can't close the panel. Remove the [attached_device] first.</span>")
+
+	else if(istype(I, /obj/item/weapon/wirecutters))
+		if(riggedstate == 3)
```

Comment:
For the purposes of fun and reducing repetition I'm going to request you add a `/datum/wires` to this and replace the disarming step with that.